sunpy/util/config.py

```diff
@@ -24,40 +36,59 @@ def load_config():
     # Specify the working directory as a default so that the user's home
     # directory can be located in an OS-independent manner
     if not config.has_option('general', 'working_dir'):
-        config.set('general', 'working_dir', os.path.join(_get_home(), "sunpy"))
+        config.set('general', 'working_dir', str(Path.home() / "sunpy"))
 
     # Specify the database url as a default so that the user's home
     # directory can be located in an OS-independent manner
     if not config.has_option('database', 'url'):
-        config.set('database', 'url', "sqlite:///" + os.path.join(
-            _get_home(), "sunpy/sunpydb.sqlite"))
+        config.set('database', 'url', "sqlite:///" + str(Path.home() / "sunpy" / "sunpydb.sqlite"))
 
-    # Use absolute filepaths and adjust OS-dependent paths as needed
-    filepaths = [
-        ('downloads', 'download_dir'),
-        ('downloads', 'sample_dir')
-    ]
-    _fix_filepaths(config, filepaths)
+    working_dir = Path(config.get('general', 'working_dir'))
+    sample_dir = Path(config.get('downloads', 'sample_dir'))
+    download_dir = Path(config.get('downloads', 'download_dir'))
+    config.set('downloads', 'sample_dir', str((working_dir / sample_dir).expanduser().resolve()))
+    config.set('downloads', 'download_dir', str((working_dir / download_dir).expanduser().resolve()))
```

Comment:
Can't config use `Path` directly?

Reply:
That is a good question and one I did not plan to test out until we plan to move to `Path` in more files.